src/grpc/mod.rs

```diff
@@ -522,7 +526,9 @@ pub async fn run_grpc_bidi_stream(
                                     Some(core_response::Payload::InstanceInfo(response_payload))
                                 }
                                 Err(err) => {
-                                    error!("Instance info error {err}");
```

Comment:
I'd leave it here just in case we forget to print an error in the handler in the future. Better to have an error logged twice than not at all and most prod instances work @ info log level. It also logs the final error as tonic Status after the conversions from base error.